README.md

```diff
@@ -1021,6 +1021,53 @@ Bazen geçici değişkenler, sonraki kod satırlarında da kullanılıyor olabil
 
 ### Replace Temp with Query
 
+#### Problem
+
+Bir ifadenin sonucunu daha sonra kullanmak üzere lokal bir değişkende tutulması.
+
+<details>
+  <summary>C#</summary>
+
+```csharp
+  double getTotalCartPrice()
+  {
+      double totalPrice = productQuantity * productPrice;
+
+      return totalPrice < 200 ? totalPrice + 80 : totalPrice;
+  }
+```
+</details>
+
+#### Çözüm
+
+İfadenin sonucunu döndğren bir metod oluşturun. Değişken kullandığınız yerde metodu çağırın.
```

Comment:
döndğren -> döndüren  | metod -> metot

Reply:
> metot kelimesi ek alırsa metodu olur fakat, normal halinde metot diye yazılır

Haklısınız pek dikkat etmemişim. Düzeltme için teşekkür ederim.